### PR TITLE
Blog post - Prague 2018 scheduling and inclusivity

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- The weather from the end of September takes a turn for the worse in Prague (and in most of Europe), and we want attendees to experience the city and not just the conference venue.
+- The global core team's yearly offsite and retrospective happens immediately after the Prague conference, and we need to allow for enough time before the end of the calendar year to wrap-up and start planning for the following year.
+
+What happened this year
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When we looked at the calendar to choose the dates for the 2018 conference, we saw that the Jewish New Year holidays span most of of September.
+This does not occur every year, but this year it was extremely difficult to decide on dates in September, since no matter which date we chose, it would conflict with one of the holidays.
+
+When we weighed this problem against the key factors that we use to determine the dates each year, we knew that had to make a decision that is going to exclude some people from being able to attend.
```
Comment (plaindocs) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: missing "we"

and the tense is a little off. Maybe 

> When we weighed this problem against the key factors that affect the dates each year, we knew that we had to make a decision that was going to exclude some people from being able to attend.

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+We owe it to our community to adhere to our philosophy of inclusivity in how we run our events, and if we are unable to choose the most inclusive option for our attendees, we should communicate about it promptly and transparently.
+In this case, we didn't do so, which caused portions of our community to feel as if they might have been overlooked, or worse - not welcome at our events.
+
+This was absolutely not our intention, and we are sorry that our actions had this impact on parts of our community.
+
+What will we do better in the future
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Firstly, we want to acknowledge that in 2019, the Jewish New Year holidays period begins in October, so there is absolutely no expected conflict with any dates we might choose in September.
+That said, we cannot guarantee that any future conference years (Prague, Portland, or otherwise) will not conflict with anything ever again, as this is a long-term venture and cannot predict what the far future will hold.
```
Comment (plaindocs) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: maybe elsewhere instead of otherwise

Comment (plaindocs) [reply docs/conf/prague/2018/news/scheduling-inclusivity.rst]: Maybe delete or reword 

> and cannot predict what the far future will hold

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+What we can promise is to improve our transparency around our scheduling practices, so that in case we must make another difficult decision in the future, we will inform you, the community, our documentarian family, of the process and considerations involved.
+
+Thank you for being you
+~~~~~~~~~~~~~~~~~~~~~~~
+
+We love Write the Docs because it's where documentarians from all walks of life should feel welcome.
```
Comment (plaindocs) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: remove "should" ?

**Review by plaindocs — COMMENTED** [review]
That reads great! Great job

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+When we looked at the calendar to choose the dates for the 2018 conference, we saw that the Jewish New Year holidays span most  of September.
+This does not occur every year, but this year it was extremely difficult to decide on dates in September, since no matter which date we chose, it would conflict with one of the holidays.
+
+When we weighed this problem against the key factors that we use to determine the dates each year, we knew that we had to make a decision that would exclude some people from being able to attend.
+We weren't happy about this decision, but if we moved the conference to any other month, or even to the first weekend of September, we would run the risk of excluding different groups of people, or not being able to run the conference at all this year due to organizational meltdowns.
+
+What could we have done better
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aside from the risks involved with choosing different dates or canceling the Prague conference altogether, there wasn't much we could have done with regards to the scheduling this year.
```
Comment (kelly-o) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: I think we should cut this. It's for sure true, but starting off the 'what we could've done better' section with a 'Well, we really didn't have a choice' sentence feels rough. The next sentence is what this section is all about, so I think we should cut right to that chase.

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+What we realized following the feedback from the community, was that we failed to communicate our decision in a transparent and empathetic way at the time of annoucning the dates.
```
Comment (kelly-o) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: so if this is the first sentence in the section, I'd just say 'We failed to communicate the context around our decision...'

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst (hunk 2 above)
Comment (kelly-o) [reply docs/conf/prague/2018/news/scheduling-inclusivity.rst]: seconded – I think I'd say, 'as this is a long-term venture, there are sure to be more difficult decisions we have to make, down the line.' or something

**Review by kelly-o — COMMENTED** [review]
reads great! nice work!

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst (hunk 4 above)
Comment (idan) [reply docs/conf/prague/2018/news/scheduling-inclusivity.rst]: Agree. The issue here wasn't the booking, it was the communication. Canceling isn't the core thing, cut this.

File: docs/conf/prague/2018/news/scheduling-inclusivity.rst [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+Simply put - this blog post should have been published months ago.
+
+After we announced the dates, we received and handled several inquires by email or Slack messages from community members who questioned the date selection and asked to reschedule, but by the time we announced the dates we already had the venue contracts signed so it wasn't possible to make a change.
+
+Regardless of the actual scheduling problem, the fact that we received inquiries should have been a warning bell for us, as we saw in the past that for every person who writes to us with a question, there must be dozens more who wouldn't feel comfortable writing to us.
+We shouldn't have kept our responses to private channels, and we should have made a public accountability statement to address this much sooner.
+
+We owe it to our community to adhere to our philosophy of inclusivity in how we run our events, and if we are unable to choose the most inclusive option for our attendees, we should communicate about it promptly and transparently.
+In this case, we didn't do so, which caused portions of our community to feel as if they might have been overlooked, or worse - not welcome at our events.
```
Comment (idan) [inline docs/conf/prague/2018/news/scheduling-inclusivity.rst]: This is the crux of this post. I'd consider emphasizing it for skimmers.

**Review by idan — APPROVED** [review]
💪🏻👏🏻

This is excellent.

The only nit I have to pick is with the future section. There will always be conflicts, maybe say that you'll try to take past conflicts into account so as to prevent the same group(s) of people from being excluded on running years. I think that's a more concrete and positive statement than "we can't predict the future". But regardless, this post is deserving of emulation. 

Thank you so, so much for caring about this. I love my community.

**Pull request merged**